charts/celestia-node/values.yaml

```diff
@@ -74,6 +74,12 @@ diagnosticMode:
 ## %%MAIN_CONTAINER/POD_DESCRIPTION%%
 ##
 node:
+  ## Select from: bridge, full, light
+  ## Valid values:
+  ## - bridge
+  ## - full
+  ## - light:
```

Comment:
We need to add some validation to ensure that the user sets one of these values.
One idea would be to use this: https://github.com/celestiaorg/helm-charts/blob/ac54d08c18c96ad5c87796c34b8d9e008f1f2fa8/charts/celestia-node/templates/_helpers.tpl#L46

Reply:
great idea!